src/napoleon_navigation_rosnode.cpp

```diff
@@ -35,6 +35,7 @@
 #include <nav_msgs/Path.h>
 
 #include <std_msgs/Bool.h>
+#include <std_msgs/Empty.h>
```

Comment:
it's probably better to make this Bool so that it's more explicit what should be done. Otherwise whoever is calling this has to remember if it's already paused or not.